cli_tools_tests/e2e/gce_image_import_export/test_suites/import/cli_tests.go

```diff
@@ -316,41 +316,41 @@ func runImageImportWithSubnetWithoutNetworkSpecified(ctx context.Context, testCa
 	logger *log.Logger, testProjectConfig *testconfig.Project, testType e2e.CLITestType) {
 
 	suffix := path.RandString(5)
-	imageName := "e2e-test-image-import-subnet-" + suffix
-	region, _ := paramhelper.GetRegion(testProjectConfig.TestZone)
+	destinationImage := "e2e-test-image-import-subnet-" + suffix
 
+	// This project doesn't have a 'default' network. Import will fail if a worker
+	// isn't configured to use the custom network and subnet.
+	project := "compute-image-test-custom-vpc"
+	subnet := "regions/us-central1/subnetworks/unrestricted-egress"
+	zone := "us-central1-a"
 	argsMap := map[e2e.CLITestType][]string{
-		e2e.Wrapper: {"-client_id=e2e", fmt.Sprintf("-project=%v", testProjectConfig.TestProjectID),
-			fmt.Sprintf("-image_name=%s", imageName), "-data_disk",
-			fmt.Sprintf("-source_file=gs://%v-test-image/image-file-10g-vmdk", testProjectConfig.TestProjectID),
-			fmt.Sprintf("-subnet=https://www.googleapis.com/compute/v1/projects/%v/regions/%v/subnetworks/%v-subnet-1",
-				testProjectConfig.TestProjectID, region, testProjectConfig.TestProjectID),
-			fmt.Sprintf("-zone=%v", testProjectConfig.TestZone),
+		e2e.Wrapper: {"-client_id=e2e", fmt.Sprintf("-project=%v", project),
+			fmt.Sprintf("-image_name=%s", destinationImage),
+			"-source_image=projects/compute-image-tools-test/global/images/debian-9-translate",
```

Comment:
Switching to import from image (vs from a file) skips testing the legacy inflater in custom network scenario.

Reply:
That is also what I want to say - the inflater has more steps to use network/subnetwork.

Reply:
good call. Updated it to use a VMDK.